modules/services/dunst.nix

```diff
@@ -61,7 +64,7 @@ in
       };
 
       settings = mkOption {
-        type = types.attrsOf types.attrs;
+        type = with types; attrsOf (attrsOf eitherStrBoolIntList);
```

Comment:
why do you have two attrsOf here?

Reply:
Since it originally was `types.attrsOf types.attrs` it is an attribute set of attribute sets.